Casks/l/ltspice.rb

```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url :homepage
-    regex(/for\s+MacOS[\s\S]*Version\s+v?(\d+(?:\.\d+)+)/i)
+    regex(/for\s+MacOS[\s\S]*?Version\s+v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/for\s+MacOS.*?Version\s+v?(\d+(?:\.\d+)+)/im)
```

While we're at it, `[\s\S]` encompasses basically everything (it's "whitespace and non-whitespace"), so we can simply use `.` instead (though we have to use the `m` flag to allow `.` to match across lines). I agree that making this non-greedy is the right approach 👍